_inc/lib/class-jetpack-recommendations.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Utilities related to the Jetpack Recommendations
+ *
+ * @package jetpack
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+
+/**
+ * Contains utilities related to the Jetpack Recommendations.
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Jetpack_Recommendations class
+ */
+class Jetpack_Recommendations {
+	/**
+	 * Returns a boolean indicating if the Jetpack Recommendations are enabled.
+	 *
+	 * @since 9.3.0
+	 *
+	 * @return bool
+	 */
+	public static function is_enabled() {
+		$recommendations_enabled = Jetpack_Options::get_option( 'recommendations_enabled', null );
+
+		// If the option is already set, just return the cached value.
+		// Otherwise calculate it and store it before returning it.
+		if ( null !== $recommendations_enabled ) {
+			return $recommendations_enabled;
+		}
+
+		$blog_id = Jetpack_Options::get_option( 'id' );
+		if ( ! $blog_id ) {
```

Comment:
I would recommend using `Connection\Manager::is_connected()` here. A site might have a lost blog ID info in the database but still not being properly connected.

Also, we are dropping the term "registered" in favor of "connected" to be consistent. In WPCOM, we never use "registered".